packages/extension-ordered-list/src/ordered-list.ts

```diff
@@ -78,13 +88,23 @@ export const OrderedList = Node.create<OrderedListOptions>({
   },
 
   addInputRules() {
-    return [
-      wrappingInputRule({
+    let inputRule = wrappingInputRule({
+      find: inputRegex,
+      type: this.type,
+    })
+
+    if (this.options.keepMarks || this.options.keepAttributes) {
+      inputRule = wrappingInputRule({
         find: inputRegex,
         type: this.type,
-        getAttributes: match => ({ start: +match[1] }),
-        joinPredicate: (match, node) => node.childCount + node.attrs.start === +match[1],
```

Comment:
I believe these removed lines broke support for creating ordered lists with a `start` offset and have opened an issue with more details: https://github.com/ueberdosis/tiptap/issues/3831

Reply:
@javan great find, I have raised #3833 to resolve this